hazelcast/src/test/java/com/hazelcast/replicatedmap/ReplicatedMapTest.java

```diff
@@ -557,6 +559,64 @@ public void testRemoveBinaryDelayDefault() throws Exception {
         testRemove(buildConfig(InMemoryFormat.BINARY, ReplicatedMapConfig.DEFAULT_REPLICATION_DELAY_MILLIS));
     }
 
+    @Test
+    public void testContainsKey_returnsFalse_onRemovedKeys() throws Exception {
+        HazelcastInstance node = createHazelcastInstance();
+        ReplicatedMap<Integer, Integer> map = node.getReplicatedMap("default");
```

Comment:
There should not be any potential conflict, all instances are using mocked networks.